### PR TITLE
Fixed #24457 -- Translated messages output by runserver

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: django/core/management/commands/runserver.py [hunk 1]
```diff
@@ -140,15 +133,15 @@ def inner_run(self, *args, **options):
         except socket.error as e:
             # Use helpful error messages instead of ugly tracebacks.
             ERRORS = {
-                errno.EACCES: "You don't have permission to access that port.",
-                errno.EADDRINUSE: "That port is already in use.",
-                errno.EADDRNOTAVAIL: "That IP address can't be assigned-to.",
+                errno.EACCES: _("You don't have permission to access that port."),
+                errno.EADDRINUSE: _("That port is already in use."),
+                errno.EADDRNOTAVAIL: _("That IP address can't be assigned-to."),
```
Comment (timgraham) [inline django/core/management/commands/runserver.py]: you can replace the dash in this message with a space

**Comment by timgraham** [comment]
Looks good. I couldn't see if checks framework messages are localized or not. Just an idea for a follow up if you didn't think of it.

**Comment by claudep** [comment]
Just added a second commit (to be merged) for date localization.

**Comment by timgraham** [comment]
Looks okay to me. Not sure if Aymeric's comment on the ticket merits additional discussion.

**Pull request closed without merging**